src/query-builder/QueryBuilder.ts

```diff
@@ -209,6 +209,13 @@ export abstract class QueryBuilder<Entity extends ObjectLiteral> {
      */
     update(): UpdateQueryBuilder<Entity>
 
+    /**
+     * Creates UPDATE query for the given entity.
+     */
+    update<Entity extends ObjectLiteral>(
```

Comment:
I'm surprised it's not already covered by the definition below:

https://github.com/typeorm/typeorm/pull/11296/files#diff-5725f0aa91cd5f33f502f605bcbc2ef2549c9f92a9ca1c984c03c2b8a80d73a5L227-L233